### PR TITLE
Allowing cache to work with multiple databases

### PR DESCRIPTION
**Pull request by ryakh**

### CODE REVIEW

File: lib/stockpile.rb [hunk 1]
```diff
@@ -36,16 +40,17 @@
 # = Stockpile
 #
 # Simple cache with Redis as a backend and a built in cache-stampede
-# protection. For more information on general usage consider consulting
-# README.md file.
+# protection and multiple Redid database support. For more information on
```
Comment (awj) [inline lib/stockpile.rb]:
```suggestion
# protection and multiple Redis database support. For more information on
```

File: lib/stockpile.rb [hunk 2]
```diff
@@ -59,7 +64,9 @@ def configuration
     @configuration ||= Configuration.new
   end
 
-  # API to configure cache dynamically during runtime.
+  # API to configure cache dynamically during runtime. Running dynamic
+  # configuration # will rebuild connection pools releasing existing
```
Comment (awj) [inline lib/stockpile.rb]:
```suggestion
  # configuration will rebuild connection pools releasing existing
```

**Comment by ryakh** [comment]
@awj this is a non breaking opt-in change. Everything will continue working as is with no additional actions required thus it's only a minor version.

**Comment by awj** [comment]
Sweet, then :shipit:

**Pull request merged**